apps/g35/g35.star

```diff
@@ -0,0 +1,146 @@
+"""
+Applet: G35
+Summary: Clinton-Washington G
+Description: Train arrival times for the Clinton-Washington station of NYC's G train.
+Author: samdotdesign
+"""
+
+load("cache.star", "cache")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("time.star", "time")
+
+MTA_API_URL = "https://mta-api-ochre.vercel.app/by-id/G35"
+CACHE_TTL = 600  # Cache data for 10 minutes
+
+# Adjustable padding values
+DIRECTION_PADDING = dict(top = 2, right = 0, bottom = 1, left = 2)
+TIME_PADDING = dict(top = -2, right = 3, bottom = 0, left = 0)
+TIMES_ROW_PADDING = dict(top = 0, right = 0, bottom = 0, left = 2)
+
+def main():
+    cached_data = cache.get("train_data")
+    api_error = None
+
+    rep = http.get(MTA_API_URL, ttl_seconds = 60)
+    if rep.status_code != 200:
+        api_error = "HTTP Error: {}".format(rep.status_code)
+    else:
+        data = rep.json()
+        if validate_data(data):
+            cache.set("train_data", json.encode(data), ttl_seconds = CACHE_TTL)
```

Comment:
Instead of usingg the cache module, you can pass `ttl_seconds` directly to `http.get()`. Would be a good change to get in before merging, as we're trying to keep down usage of the cache module itself.

Reply:
Thanks! Fixed.